Jenkinsfile

```diff
@@ -0,0 +1,8 @@
+#!groovy
+@Library('rspace-shared') _ 
+// builds DigitalCommonsData-adapter project
+genericJavaLibBuild("emails":"dev@researchspace.com",
+ "branch":"${BRANCH_NAME}",
+  "jdk":"OPEN-JDK-11",
+  "maven":"maven3.8.1",
+  "jvmArgs":"-Djava-vendor=openjdk -Djava-version=11")
```

Comment:
specifying java 11 shouldn't be necessary - if params are not provided, then[ Java 17 would be used by default](https://github.com/ResearchSpace-ELN/jenkins-pipeline-shared/blob/master/vars/genericJavaLibBuild.groovy), which is what we want here. Compare with [rspace-figshare-adapter](https://github.com/rspace-os/rspace-figshare-adapter/blob/main/Jenkinsfile).

Reply:
changed